decentraland/unity-explorer
# Feat: used emotes analytics

## What does this PR change?

- add ECS System for triggered emotes (by player or by scene)
- add Decorator to handle equipped emote

## How to test the changes?

1. Launch the explorer
2. Verify "happy path" is not broken (and emotes are playing as previously - by player and by scene)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
